### ISSUE TITLE
Accessibility: missing the aria-required attribute in the comment field 

### ISSUE THREAD

**Issue by tsv2013** [surveyjs/survey-library#2235]
See the https://surveyjs.answerdesk.io/ticket/details/t4438/accessibility-add-required-attribute-to-comment-field thread for more retails
